provider/bigquery.go

```diff
@@ -606,7 +606,10 @@ func (it *bqFeatureIterator) Next() bool {
 
 	var currValue ResourceRecord
 	valueColType := it.iter.Schema[1].Type
-	currValue.Entity = rowValue[0].(string)
+	if err := currValue.SetEntity(rowValue[0]); err != nil {
```

Comment:
you should be able to do 
```
currValue.Entity, ok := rowValue[0].(string); if !ok {
}
```

Reply:
@sdreyer , `SetEntity` returns `error`, so I'm not sure if you're asking me to change the return type. Happy to make this change.